### ISSUE TITLE
Explosives won't work

### ISSUE THREAD

**Issue by XFlawless1** [Southclaw/ScavengeSurvive#270]
[22:42:24] [GetItemArrayDataAtCell] ERROR: Cell 0 exceeds item type (124) max of 0.
[22:43:10] [GetItemArrayDataAtCell] ERROR: Cell 0 exceeds item type (129) max of 0.
[22:43:50] [GetItemArrayDataAtCell] ERROR: Cell 0 exceeds item type (125) max of 0.
[22:44:55] [GetItemArrayDataAtCell] ERROR: Cell 0 exceeds item type (220) max of 0.

[22:54:56] [GetItemArrayDataAtCell] ERROR: Cell 0 exceeds item type (170) max of 0.
Latest version
BUILD 1085
Latest SIF aswell.

**Comment by Southclaw** [comment]
None of those items are explosives, do you have any more information? I don't think there's any code that sets the extradata for those itemtypes anyway.

**Comment by XFlawless1** [comment]
I did prefer trying planting a bomb any bomb.

**Comment by XFlawless1** [comment]
No replies from server side,
[11:22:57] [COMMAND] [XFlawless]: /additem timed
[11:23:05] [LOGOUT] Ghazzawi logged out at -2421.5, -2327.8, 18.1 (276.8) Logged In: 1 Alive: 1 Knocked Out: 0
[11:23:05] [PART] Ghazzawi (lost connection)
[11:23:06] [VEHICLE] BilboTeaBaggins entered vehicle as driver 3 (Huntley) at 430.529479, 2522.018310, 16.487106
[11:23:06] [SAVE] Vehicle 3 (U) 0 items: Huntley for BilboTeaBaggins at 430.52, 2522.01, 16.48
[11:23:14] [COMMAND] [XFlawless]: /additem timed

**Comment by trojan-sas** [comment]
he mean its not shown as explosive item i think that when u press f to plant nothing happened

**Comment by XFlawless1** [comment]
Well I posted when I planted I am not so dumb BTW.

**Comment by Southclaw** [comment]
Tested and found the issue (I didn't really understand the bug report, sorry).

The explosion list on SA:MP wiki is wrong and some of the explosion types I used in the explosion module presets were too small (the radius parameter for CreateExplosion does absolutely nothing, set it to 0.0 and the explosion still has the same damage range).

Fixed for next update.

Noticed no issues with interaction (holding/pressing F to plant - all works fine).

Noticed no issues with defences (TNT does 2 damage, IED does 1, as usual).
